a_bot_gripper_controller2:
  default_controllers:
    - "a_bot_gripper_controller2/gripper_action"

  gripper_action:
    type: "aist_robotiq2f85_controller/Robotiq85GripperController"

    l_gripper_joint: "robotiq_85_left_knuckle_joint"
    r_gripper_joint: "robotiq_85_right_knuckle_joint"

    command_topic: "/a_bot_gripper_controller/command"
    status_topic: "/a_bot_gripper_controller/status"

    # Feedback rate in Hz
    publish_rate: 100
    min_gap_counts: 224
    # Gripper width in meters [0, 0.085]
    min_gap:    0.0
    max_gap:    0.085
    # Gripper speed in m/s [0.013, 0.1]
    min_speed:  0.013
    max_speed:  0.1
    # Gripper force in N. [40, 100]
    min_force:  40.0
    max_force:  100.0

  robotiq_85_left_knuckle_joint:
    position:
      p: 5000.0
      d: 0.0
      i: 0.0
      i_clamp: 0.0
    velocity:
      p: 0.0
      d: 0.0
      i: 0.0
      i_clamp: 0.0

  robotiq_85_right_knuckle_joint:
    position:
      p: 5000.0
      d: 0.0
      i: 0.0
      i_clamp: 0.0
    velocity:
      p: 0.0
      d: 0.0
      i: 0.0
      i_clamp: 0.0

